content/en/docs/how-tos/onboarding-a-new-component.md

```diff
@@ -238,15 +240,17 @@ When container images are declared as release artifacts for a repository in the
 a synthetic `[images]` target is available for CI Operator execution that will simply build all
 release images. In order for the container images built from your repository to be published, the
 Prow job generator will configure a Prow postsubmit job that uses the CI Operator `--target=[images]` and `--promote` flags.
-Information on how to publish images to an external registry can be found in a [separate document.](/docs/how-tos/mirroring-to-quay/).
+Information on how to publish images to an external registry can be found in a [separate document](/docs/how-tos/mirroring-to-quay/).
 
-### Product builds and becoming part of an OpenShift release image
+### Product builds and becoming part of an OpenShift release
 
-Some images become part of the OpenShift release image because they are core to the platform. To be part of the release
-image, there are additional requirements (beyond those described in the previous section).
+Some images become part of the OpenShift release because they are core to the platform. To be part of the release, there are additional requirements (beyond those described in the previous section).
+Adding images to releases is a significant addition, and requires [an enhancement][enhancements] to explain why the new functionality is important, and why it should be delivered via OpenShift releases (instead of via [an OLM-installed operator][operator-lifecycle-manager]).
+It also requires [the approval of image names for mirroring](/docs/how-tos/mirroring-to-quay/#mirroring-images).
```

Comment:
to be clear any image that gets mirrored requires this approval, whether the image is in the release/payload or not